### ISSUE TITLE
Incorrect output directory for eclipse projects on Windows for 1.2.12

### ISSUE THREAD

**Issue by nedtwigg** [palantir/gradle-processors#70]
- On unix machines, this project generates eclipse projects whose output source is `generated/java`
- On windows machines, the output source is instead just `generatedjava`, but the plugin itself puts the code in `generated/java`, so there are two copies of the output source.

The cause is that the `.settings/org.eclipse.jdt.apt.core.prefs` has the following error:

```
org.eclipse.jdt.apt.genSrcDir=generated\java   <-- what it does have
org.eclipse.jdt.apt.genSrcDir=generated\\java  <-- what it should have
```

One fix is to [escape the argument when it is passed](https://github.com/palantir/gradle-processors/blob/8b6dc7152e1774a133a496070b57c022891c9a6e/src/main/groovy/org/inferred/gradle/ProcessorsPlugin.groovy#L66), and the other is to fix it [in the template engine](https://github.com/palantir/gradle-processors/blob/8b6dc7152e1774a133a496070b57c022891c9a6e/src/main/groovy/org/inferred/gradle/ProcessorsPlugin.groovy#L237).

Easiest thing is to escape the argument with a simple `.replace("\\", "\\\\")`.

**Comment by nedtwigg** [comment]
Workaround until a fix is released:

```groovy
// workaround until https://github.com/palantir/gradle-processors/issues/70 is merged
tasks.eclipse.doLast {
    ant.replace(file: '.settings/org.eclipse.jdt.apt.core.prefs', token: 'generated\\java', value: 'generated\\\\java')
}
```

**Comment by chrisalice** [comment]
Released as [v1.2.14](https://github.com/palantir/gradle-processors/releases/tag/v1.2.14), thanks!
